tests/test_firefox_urls.py

```diff
@@ -37,6 +37,11 @@ class TestFirefoxURLs(object):
         '/firefox/tiles/',
         '/firefox/unsupported-systems/',
         '/firefox/unsupported/EOL/',
+        # Legacy URLs
+        '/firefox/start/central.html',
+        '/firefox/en-US/firefox/whatsnew/',
+        '/firefox/panorama/',
+        'firefox/sync/firstrun.html',
```

Comment:
testing manually "/firstrun.hmtl/" gives a 404 while the path as written redirects correctly